server/sonar-web/src/main/js/apps/organizations/components/OrganizationGroupCheckbox.js

```diff
@@ -20,18 +20,20 @@
 //@flow
 import React from 'react';
 import Checkbox from '../../../components/controls/Checkbox';
-import type { OrgGroup } from '../../../store/organizations/duck';
+/*:: import type { OrgGroup } from '../../../store/organizations/duck'; */
 
+/*::
 type Props = {
   group: OrgGroup,
   checked: boolean,
   onCheck: (string, boolean) => void
 };
+*/
 
 export default class OrganizationGroupCheckbox extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  onCheck = (checked: boolean) => {
+  onCheck = (checked /*: boolean */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)